cli/tests/install.bats

```diff
@@ -534,3 +534,41 @@ EOF
   assert_success
   assert_output "$vim_store_path/bin/vim"
 }
+
+# ---------------------------------------------------------------------------- #
+
+@test "'flox install' info message when overriding included package" {
+  # This will be an included environment
+  "$FLOX_BIN" init -d included
+
+  # Install a package that will be overridden by the composer later
+  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
+    "$FLOX_BIN" install -d included hello
+
+  # Create the composing environment
+  "$FLOX_BIN" init -n "composer"
+
+  # Add the include to the manifest
+  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
+  "$FLOX_BIN" edit -f - <<- EOF
+version = 1
+
+[include]
+environments = [
+  { dir = "included" }
+]
+
+[options]
+systems = ["aarch64-darwin", "x86_64-darwin", "aarch64-linux", "x86_64-linux"]
```

Comment:
nit: I think leaving this out keeps our test manifests ever so slightly more readable
```suggestion
```